PyroApp/source/Components/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { View, Text, StyleSheet, Pressable } from 'react-native'
+
+const CustomButton = ({ onPress, text, type = "PRIMARY" }) => {
+    return(
+        <Pressable onPress={onPress} style={[styles.container, styles[`container_${type}`]]}>
+            <Text style={[styles.text, styles[`text_${type}`]]}> {text} </Text>
+        </Pressable>
+    );
+};
+
+const styles = StyleSheet.create({
+    container: {
+        width: '100%',
+
+        padding: 15,
+        marginVertical: 5,
+
+        alignItems: 'center',
+        borderRadius: 5,
+    },
+    container_PRIMARY: {
+        backgroundColor: '#3B71F3',
+    },
+    container_TERTIARY: {},
+
+    text: {
+        fontWeight: 'bold',
+        color: 'white',
+    },
+    text_Primary:{},
+    text_TERTIARY: {
+        color: 'grey',
+    },
+})
```

Comment:
de manière générale je n'aime pas trop le CSS-in-js. Je trouve que ça ne permet pas d'avoir un style partagé, uniforme et que cela génère souvent des bugs d'alignement et autre. Je te suggère de regarder du côté de Tailwind ou a minima de créer des styles partagés